### DISCUSSION TITLE
docker部署后客户端无法连接到网络

### DISCUSSION THREAD

**Discussion by Cryolitia** [1715173329/UnblockNeteaseMusic#685] [Q&A]
硬件：树莓派4B 8G版
操作系统： Manjaro-ARM 22.04
内核：aarch64 Linux 5.15.36-1-MANJARO-ARM-RPI
Docker版本：Docker version 20.10.14, build a224086349

使用命令：`sudo docker run -p 1149:1149 -e ENABLE_FLAC=true -e LOG_LEVEL=debug pan93412/unblock-netease-music-enhanced -p 1149`

问题：网页`netease.163.com`无法打开，提示连接被重置；客户端提示网络不给力。
![image](https://user-images.githubusercontent.com/23723294/168148479-4bbef159-fa72-4287-a18b-14335182e45b.png)
![image](https://user-images.githubusercontent.com/23723294/168148507-4cd6cd55-edd5-4c71-897b-6c3d0ec32495.png)

日志：
![image](https://user-images.githubusercontent.com/23723294/168148556-31d42eed-288c-486a-aedb-dd894978fa14.png)

**Comment by XiaChuerwu** [comment]
docker run pan93412/unblock-netease-music-enhanced -o bilibili 部署到服务器docker也是同样无法使用

**Comment by XiaChuerwu** [comment]
![image](https://user-images.githubusercontent.com/93534259/168196383-db2ad423-1e51-4676-9fb5-83dcf67047fa.png)

**Comment by XiaChuerwu** [comment]
![image](https://user-images.githubusercontent.com/93534259/168196441-28dc26d4-5e18-46bf-9089-3cac660bfb32.png)

**Comment by XiaChuerwu** [comment]
已经放行了8080 8081端口

**Comment by 1715173329** [comment]
如果不熟悉 docker 可以直接从源码运行。

**Reply by Cryolitia** [reply]
> 如果不熟悉 docker 可以直接从源码运行。

clone源码，release里的二进制，aur源里的都试过了，都不能用，一样的问题

**Reply by 1715173329** [reply]
需要设置 endpoint，需要信任证书，见 readme。

**Reply by Cryolitia** [reply]
> 需要设置 endpoint，需要信任证书，见 readme。

已经信任证书了，endpoint不知道是干什么的怎么用，readme里endpoint的介绍约等于没有

**Reply by 1715173329** [reply]
-e https://music.163.com
